src/display/api.js

```diff
@@ -717,6 +717,8 @@ var PDFDocumentProxy = (function PDFDocumentProxyClosure() {
  * @property {Object} canvasFactory - (optional) The factory that will be used
  *                    when creating canvases. The default value is
  *                    {DOMCanvasFactory}.
+ * @property {string} backgroundColor - Background color to use for the canvas.
+ *                    The default value is 'rgb(255,255,255)'
```

Comment:
Please mark this as `(optional)`, refer e.g. to the `canvasFactory` just above. Also, the `The default value is 'rgb(255,255,255)'` sentence is missing a period at the end.

Finally, when all review comments have been addressed, please squash the commits into one; see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.